.travis.yml

```diff
@@ -31,6 +31,9 @@ install:
   - ./webdis &
   - popd
   - travis_retry pip install . --no-binary redbiom
+  # staring nginx
+  - mkdir /home/travis/miniconda3/envs/qiita/var/run/nginx/
```

Comment:
Not sure why are we putting the NGINX directory under Qiita's Conda environment. Is that just for document root purposes?

Reply:
Good question, note that this due to how conda adds nginx and the folders it expects to have to create logs/etc; will add it to the instructions